packages/origin-backend/migrations/initial.sql

```diff
@@ -163,7 +163,9 @@ CREATE TABLE public."user" (
     notifications boolean,
     "autoPublish" text,
     rights integer DEFAULT 0 NOT NULL,
-    "organizationId" integer
+    "organizationId" integer,
+    "status" integer DEFAULT 0 NOT NULL,
```

Comment:
This should be removed, initial.sql is not meant to be edited, all changes are handled using migrations/* files